### ISSUE TITLE
perception verb + infinitive verb: ccomp vs. xcomp

### ISSUE THREAD

**Issue by nschneid** [amir-zeldes/gum#154]
https://universal.grew.fr/?custom=647bd7fa3c501 - xcomp is correct, right?

cf. UniversalDependencies/UD_English-EWT#292

**Comment by amir-zeldes** [comment]
Hm, I'm not sure what's right... I think the ccomps are intentional in any case. My memory on this is a big dim, but I think in SD EWT, 'see' and 'hear' had this ccomp behavior which I interpreted to be a reading of the ECM small-clause analysis, where "I saw Kim dancing" is me seeing the whole event "Kim is dancing", and not actually seeing Kim. I will readily admit I don't see a big difference between this and causative xcomp of the type "make Kim dance", but I think that's the way it was in EWT back then and we just emulated it. I would be open to using xcomp throughout, but maybe we can get some historical context from people involved in the EWT -> SD conversion (unless I'm just hallucinating all of this)
